connector/gitea/gitea.go

```diff
@@ -264,3 +417,10 @@ func (c *giteaConnector) user(ctx context.Context, client *http.Client) (giteaUs
 	}
 	return u, nil
 }
+
+// groupsRequired returns whether dex requires GitHub's 'read:org' scope. Dex
```

Comment:
I think the comment was just copied.
```suggestion
// groupsRequired returns whether dex requires Gitea 'read:org' scope. Dex
```

Reply:
And what about the other part of the comment? Especially the part about `org`. Is it correct?